catalog/imaging/imaging-plugin-nitf/src/test/groovy/org/codice/alliance/plugin/nitf/NitfPostProcessPluginSpec.groovy

```diff
@@ -0,0 +1,396 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p/>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p/>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.alliance.plugin.nitf
+
+import ddf.catalog.data.Metacard
+import ddf.catalog.data.impl.AttributeImpl
+import ddf.catalog.data.types.Core
+import ddf.catalog.plugin.PluginExecutionException
+import org.codice.ddf.catalog.async.data.api.internal.ProcessCreateItem
+import org.codice.ddf.catalog.async.data.api.internal.ProcessRequest
+import org.codice.ddf.catalog.async.data.api.internal.ProcessResource
+import org.codice.ddf.catalog.async.data.api.internal.ProcessUpdateItem
+import org.codice.ddf.catalog.async.data.impl.ProcessRequestImpl
+import org.codice.imaging.nitf.core.image.ImageSegment
+import org.codice.imaging.nitf.render.NitfRenderer
+import spock.lang.Specification
+import spock.lang.Unroll
+
+import java.awt.image.BufferedImage
+import java.util.function.Supplier
+
+@Unroll
```

Comment:
❓ Do we want to unroll the whole test class? Or, just the specific methods that are parameterized?